packages/sdks/nextjs-sdk/examples/app-router/app/my-applications-portal/page.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import React from 'react';
+import { ApplicationsPortal } from '@descope/nextjs-sdk'; // eslint-disable-line
+
+export default () => (
+	<div
+		style={{
+			display: 'flex',
+			flexDirection: 'column',
+			alignItems: 'center',
+			justifyContent: 'center'
+		}}
+	>
+		<h1>User Profile</h1>
+		<ApplicationsPortal
+			widgetId="applications-portal-widget"
+			onLogout={() => {
```

Comment:
do we have logout on application portal?

Reply:
Mistake